plotly/tools.py

```diff
@@ -49,6 +49,36 @@
 ALTERNATIVE_HISTNORM = 'probability'
 
 
+http_msg = (
+    "The plotly_domain and plotly_api_domain of your config file must start "
+    "with 'https', 'http'.\n If you are not using On-Prem then run the "
+    "following code to ensure your plotly_domain and plotly_api_domain start "
+    "with 'https':\n\n\n"
+    "import plotly\n"
+    "plotly.tools.set_config_file(\n"
+    "    plotly_domain='https://plot.ly',\n"
+    "    plotly_api_domain='https://api.plot.ly'\n"
+    ")\n\n\n"
+    "If you are using On-Prem then you will need to use your company's "
+    "domain and api_domain urls:\n\n\n"
+    "import plotly\n"
+    "plotly.tools.set_config_file(\n"
+    "    plotly_domain='https://plotly.your-company.com',\n"
+    "    plotly_api_domain='https://plotly.your-company.com'\n"
+    ")\n\n\n"
+    "Make sure to replace `your-company.com` with the URL of your Plotly "
+    "On-Premise server.\nSee "
+    "https://plot.ly/python/getting-started/#special-instructions-for-plotly-onpremise-users "
+    "for more help with getting started with On-Prem."
+)
+
+
+def validate_config_file(*domains):
```

Comment:
This is really just validating domains - what do you think about renaming this as `validate_domains`?

Reply:
good idea